components/MainnetHeader.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <header class="header">
```

Comment:
Why are we not reusing the existing AppHeader component?

Reply:
The design for the Mainnet page has `Technology` in the header, which is not in rest of the places

Reply:
IMO, we should request the design folks to keep the header consistent with the rest of the site.

Reply:
Repalced with AppHeader component - https://github.com/arcana-network/website/pull/139/commits/e842123998ac0d34a956960cab666bb1b7a4a4f4